README.md

```diff
@@ -3,3 +3,129 @@
 [![Hackage](https://img.shields.io/hackage/v/life-sync.svg)](https://hackage.haskell.org/package/life-sync)
 [![Build status](https://secure.travis-ci.org/kowainik/life-sync.svg)](https://travis-ci.org/kowainik/life-sync)
 [![MIT license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/kowainik/life-sync/blob/master/LICENSE)
+[![Stackage LTS](http://stackage.org/package/life-sync/badge/lts)](http://stackage.org/lts/package/life-sync)
+[![Stackage Nightly](http://stackage.org/package/life-sync/badge/nightly)](http://stackage.org/nightly/package/life-sync)
+
+`life-sync` is CLI tool that makes it easier to synchronize `dotfiles`
+repository with personal configs across multiple machines.
+
+## Motivation
+
+You might have some configuration files with different settings for your system.
+For example:
+
+1. Preferred settings for your editors (Spacemacs, Vim etc.).
+2. Useful bash aliases and other miscellaneous shell settings.
+3. Git configuration.
+
+And much more! But sometimes you start working from new fresh machine without
+your settings, like in situations below:
+
+1. You bought new PC or laptop.
+2. You might reinstall your system on your working machine.
+3. You was given new laptop at work.
+
+Every time this happens you need to walk through tedious process of copying your
+data again. It's a well-known practice to
+[store configs in `dotfiles` GitHub repository](https://dotfiles.github.io/).
+And `life-sync` makes it much easier to maintain this repository! With single
+command your can copy every file and directory from `dotfiles` repository to
+your machine. Or update your remote `dotfiles` repository after multiple local
+changs to different files.
+
+## Installation
+
+Installation process can be done with one simple command:
+
+```shell
+$ cabal install life-sync
+```
+
+or
+
+```shell
+$ stack install life-sync-1.0
+```
+
+You can turn on the bash auto-completion by running the following command:
+
+```
+$ source <(life --bash-completion-script `which life`)
+```
+
+## Usage
+
+After installing `life-sync` you need to call command `life` with specified options:
+
+```shell
```

Comment:
I think that `shell` here isn't necessary, since syntax highlighting for the help message isn't really useful.